command/scaling_policy_info.go

```diff
@@ -0,0 +1,126 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mitchellh/cli"
+	"github.com/posener/complete"
+)
+
+// Ensure ScalingPolicyInfoCommand satisfies the cli.Command interface.
+var _ cli.Command = &ScalingPolicyInfoCommand{}
+
+// ScalingPolicyListCommand implements cli.Command.
+type ScalingPolicyInfoCommand struct {
+	Meta
+}
+
+// Help satisfies the cli.Command Help function.
+func (s *ScalingPolicyInfoCommand) Help() string {
+	helpText := `
+Usage: nomad scaling policy info [options] <policy_id>
+
+  Info is used to read the specified scaling policy.
+
+General Options:
+
+  ` + generalOptionsUsage() + `
+
+Policy Info Options:
+
+  -json
+    Output the scaling policy in its JSON format.
+
+  -t
+    Format and display the scaling policy using a Go template.
+`
+	return strings.TrimSpace(helpText)
+}
+
+// Synopsis satisfies the cli.Command Synopsis function.
+func (s *ScalingPolicyInfoCommand) Synopsis() string {
+	return "Display an individual Nomad scaling policy"
+}
+
+func (s *ScalingPolicyInfoCommand) AutocompleteFlags() complete.Flags {
+	return mergeAutocompleteFlags(s.Meta.AutocompleteFlags(FlagSetClient),
+		complete.Flags{
+			"-json": complete.PredictNothing,
+			"-t":    complete.PredictAnything,
```

Comment:
Oh cool! Is this this the first command to support this?

Reply:
There are quite a few that already support this; although i'll admit I have never used them.